Makefile

```diff
@@ -500,10 +491,10 @@ proto:
 
 .PHONY: proto-install
 proto-install:
-	go get -u github.com/golang/glog
 	go install \
-		./vendor/github.com/golang/protobuf/protoc-gen-go \
-		./vendor/github.com/gogo/protobuf/protoc-gen-gogo \
-		./vendor/github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway \
-		./vendor/github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger
+		 github.com/golang/glog \
+		 github.com/golang/protobuf/protoc-gen-go \
+		 github.com/gogo/protobuf/protoc-gen-gogo \
+		 github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway \
+		 github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger
```

Comment:
How does `go install` work with modules? Is it going to install the version from go.mod?

Perhaps we can wait for @annanay25 to change the gen to use Docker image, then we won't have to do these things here.

Reply:
> How does go install work with modules? Is it going to install the version from go.mod?

That is my understanding.

> Perhaps we can wait for @annanay25 to change the gen to use Docker image, then we won't have to do these things here.

I don't want to block this PR on that. That can be done later.

Reply:
>> Is it going to install the version from go.mod?
> That is my understanding.

Can we find some evidence to that?

Reply:
For instance version of `github.com/sectioneight/md-to-godoc` is defined in `go.mod`. The never version does not work due to conflicts. I specified the older version explicitly like we had in `dep`.